prow/gerrit/client/client.go

```diff
@@ -404,6 +405,22 @@ func (c *Client) GetChange(instance, id string, addtionalFields ...string) (*Cha
 	return info, nil
 }
 
+func (c *Client) SubmitChange(instance, id string, wait bool) (*ChangeInfo, error) {
```

Comment:
Submitting the exact revision seems like a better option to avoid races: https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#submit-revision

Reply:
this is not supported yet: https://github.com/andygrunwald/go-gerrit/blob/41749a62809298f29d8bdf8b21d946853972ec44/changes_revision.go#L649.

Did a line by line comparison between Submit Revision and Submit Change, they are pretty much identical other than one thing that cause my attention:
```
 "revision 674ac754f91e64a0efb8087e59a176484bd534d1 is not current revision"
```

So "Submit Revision" technically is safer to avoid submitting a PR that just has a new revision pushed. However afaict Gerrit revokes `Verified` label when a new revision is pushed, which will also prevents the submission according to doc:
```
If the revision cannot be submitted, e.g. because the submit rule doesn’t allow submitting the revision or the revision is not the current revision, the response is “409 Conflict” and the error message is contained in the response body.
```